### DISCUSSION TITLE
How to change the Challenge Period on local network.

### DISCUSSION THREAD

**Discussion by nitant911** [ethereum-optimism/developers#487] [Application Development]
I am using the OP-Stack codebase to run the network locally, How to change the Challenge Period on local network.

**Comment by smallyunet** [comment]
`finalizationPeriodSeconds` in `getting-started.json`

**Comment by ForTheFutureNOW** [comment]
Curious too, has someone the answer?

**Comment by ghost** [comment]
@nitant911 @smallyunet @ForTheFutureNOW I have submitted an alternative solution to replace 7 days challenge widow https://github.com/ethereum-optimism/optimism/issues/8037
